spec/lib/court_data_adaptor/query/defendant/by_name_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject { described_class }
 
   let(:instance) { described_class.new(nil) }
-  let(:term) { 'josefa  theadore FRanecki  ' }
+  let(:term) { 'josefa theadore FRanecki  ' }
```

Comment:
Stripped spaces here as CP  seems to be allowing a more permissive search. Not entirely sure of the the view court data's requirements here.

Reply:
this was to test the parsers removal of whitespace

Reply:
From my testing it seemed to be pretty permissive:
eg: i could search with just `josefa` to get a bunch of results containing names:
[`Josefa Franecki` and `Josefa Dodger`]